docs/content.zh/docs/dev/table/types.md

```diff
@@ -264,22 +259,20 @@ DataTypes.VARCHAR(n)
 DataTypes.STRING()
 ```
 
-<span class="label label-danger">Attention</span> The specified maximum number of code points `n` in `DataTypes.VARCHAR(n)` must be `2,147,483,647` currently.
+<span class="label label-danger">请注意</span> 目前，`DataTypes.VARCHAR(n)` 中指定的最大指 `n` 当前必须为 `2,147,483,647`。
 {{< /tab >}}
 {{< /tabs >}}
 
-The type can be declared using `VARCHAR(n)` where `n` is the maximum number of code points. `n` must have a value
-between `1` and `2,147,483,647` (both inclusive). If no length is specified, `n` is equal to `1`.
+可以使用 `VARCHAR(n)` 声明该类型，其中 `n` 表示其最大值。`n` 的取值范围为[ `1` , `2,147,483,647` ] ，如果没有指定 `n` 的值，n 等于 1。
```

Comment:
How about translating (both inclusive) instead of using '[' and ']' characters to let this line more specific?